include/RAJA/exec-cuda/reduce_cuda.hxx

```diff
@@ -111,6 +121,38 @@ __device__ __forceinline__ double shfl_xor(double var, int laneMask)
 #define double_to_ull(x) __double_as_longlong(x)
 #endif
 
+/*!
```

Comment:
Should these be pulled out of the cuda reduction file and placed in raja_cuda.hxx instead?